components/Progress-Bar/progress-bar.js

```diff
@@ -0,0 +1,70 @@
+const msdsProgressBar = {
+  //Get the base number for the progress bar
+  parentElement: '',
+  baseNumber: 0,
+  currentStep: 0,
+  startFromOne: false,
+  init: function(parentElementId, baseNumber, customProgressionStep) {
+    this.parentElement = document.getElementById(parentElementId)
+
+    if (customProgressionStep > 0) {
+      this.currentStep = customProgressionStep
+    } else if (this.startFromOne == true) {
+      this.currentStep = 1
+    } else {
+      this.currentStep = 0
+    }
+    this.baseNumber = baseNumber
+    this.updateProgressbarDisplay()
+  },
+  updateProgressbarDisplay: function() {
```

Comment:
What is progressBarDisplay? Looks like there is some HTML that gets modified but the name does not correspond to any of it from what I can see?

Reply:
HI danny

Please see the full file and you will see the progressBarDisplay function. click on the 3 dots top right of this section.

Reply:
same as above. :)